index.html

```diff
@@ -0,0 +1,471 @@
+
+      <ul class="social_links">
+
+
+        <li><a href=""><img src="./images/Vector-2.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Linkedin-icon.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector-3.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+        <li><a href=""><img src="./images/Vector.jpg" class="logo-icons" alt="union" /></a></li>
+
+        <li><a href=""><img src="./images/Vector-4.jpg" class="logo-icons" alt="union" /></a></li>
+
+
+      </ul>
+    </section>
+
+    <section class="container1">
+      <div>
+        <div class="Pics">
+
+          <img src="./images/Snapshoot-icon.jpg" alt="Snapshoot" />
+        </div>
+
+        <div class="tonic">
+          <img src="./images/tonictitle.jpg" alt="tonic">
+        </div>
+
+        <ul class="dots">
+          <li class="canopy">
+            <a href="">
+              <img src="./images/canopytext.jpg" alt="canopy">
+            </a>
+          </li>
+
+          <li class="dot1">
+            <a href="">
+              <img src="./images/dot.jpg" alt="dot1">
+            </a>
+          </li>
+
+          <li class="backend">
+            <a href="">
+              <img src="./images/backenddev.jpg" alt="backend">
+            </a>
+          </li>
+
+          <li class="dot2">
+            <a href="">
+              <img src="./images/dot.jpg" alt="dot2">
+            </a>
+          </li>
+
+          <li class="year">
+            <a href="">
+              <img src="./images/2015.jpg" alt="year">
+            </a>
+          </li>
+
+        </ul>
+
+        <div class="content_paragraph">
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+
+        <div class="tech_list">
+          <ul class="list">
+            <li>
+              <a href="">
+                <img src="./images/htmltag.jpg" alt="html">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/csstag.jpg" alt="css">
+              </a>
+            </li>
+
+            <li>
+              <a href="">
+                <img src="./images/jstag.jpg" alt="js">
+              </a>
+            </li>
+          </ul>
+        </div>
+
+        <a href="">
+          <img class="button" src="./images/SeeProject.jpg" alt="">
+        </a>
+      </div>
+    </section>
+
+
+    <section class="container2">
+      <div>
+        <div class="Pics">
+
+          <img src="./images/multi-posts.jpg" alt="avialable" />
+        </div>
+
+        <div class="tonic">
+          <img src="./images/multipoststoriestitle.jpg" alt="multiposts">
+        </div>
+
+        <ul class="dots">
+          <li class="canopy">
+            <a href="">
+              <img src="./images/canopytext.jpg" alt="canopy">
+            </a>
+          </li>
+
```

Comment:
- [ ] Please kindly give make sure each card has a background of white with border-radius to match the original design. Kindly do this for all cards

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165337993-e47048d7-aec4-49f6-91a6-9922e39e872a.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165338184-bba06dea-6b10-46e2-bc18-b3d22b04ea7e.png)